client/html/templates/catalog/lists/items-body-standard.php

```diff
@@ -27,11 +27,28 @@
  * @category User
  * @see client/html/catalog/domains
  */
+$listTarget = $this->config( 'client/html/catalog/lists/url/target' );
+$listController = $this->config( 'client/html/catalog/lists/url/controller', 'catalog' );
+$listAction = $this->config( 'client/html/catalog/lists/url/action', 'list' );
+$listConfig = $this->config( 'client/html/catalog/lists/url/config', [] );
+$listParams = $this->get( 'listParams', [] );
 
+/** client/html/catalog/lists/infinite-scroll
+ * Enables infinite scrolling in product catalog list
+ *
+ * If set to true, products from the next page are loaded via XHR request
+ * and added to the product list when the user reaches the list bottom.
+ *
+ * @param boolean True to use infinite scrolling, false to fisable it
+ * @since 2018.10
+ * @category Developer
+ */
+$infiniteScroll = $this->config( 'client/html/catalog/lists/infinite-scroll', false );
 
+$infiniteUrl = ( $infiniteScroll && $this->get( 'listPageNext', 0 ) > $this->get( 'listPageCurr', 0 ) ) ? $this->url( $listTarget, $listController, $listAction, array( 'l_page' => $this->get( 'listPageNext' ) ) + $listParams, [], $listConfig + array( 'absoluteUri' => true ) ) : '';
```

Comment:
Do we really need `absoluteUri`?

Reply:
There was a bug in IE 9.x or so which prevented ajax calls to relative URLs in some cases. This is why I started using absolute URLs whereever possible in JS. But you're probably right, this is not relevant anymore - I will change it.